dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala

```diff
@@ -57,16 +57,7 @@ class AddressActor(owner: Address,
   private var openVolume   = Map.empty[Asset, Long]
   private val expiration   = MutableMap.empty[ByteStr, Cancellable]
 
-  private var addressMutableState = AddressWsMutableState.empty
-
-  private def mkWsAddressState(spendableBalances: Map[Asset, Long]): WsAddressState = {
-    val tradableBalance = spendableBalances |-| openVolume.filterKeys(spendableBalances.keySet)
-    WsAddressState(
-      spendableBalances.map {
-        case (a, _) => a -> WsBalances(tradableBalance.getOrElse(a, 0), openVolume.getOrElse(a, 0))
-      }
-    )
-  }
+  private var addressWsMutableState = AddressWsMutableState.empty
```

Comment:
FYI the variable is mutable, but `AddressWsMutableState` is not :D

Reply:
Hope I'm not complete dummy since I knew this